organization/class/Home.md

```diff
@@ -0,0 +1,64 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Organization/Home
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2 (Not yet)](https://img.shields.io/badge/API-Twitter%202%20(Not%20yet)-darkred?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public interface Home : Collection
+```
+
+`Home` inheriteds [`Collection`](Collection.md) and provides support for the users home timeline.
+
+| API      | Twitter 1 | Twitter 2 | Mastodon |
+| -------- | --------- | --------- | -------- |
+| Function | [`statuses/home_timeline`](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/timelines/api-reference/get-statuses-home_timeline) |  | [`timelines/home`](https://docs.joinmastodon.org/methods/timelines/) |
+
+## Properties
+
+> ### Inherited from [`Collection`](Collection.md)
+> 
+> [**`last_viewed_post`**](Collection.md#property_last_viewed_post)
+
+## Creation Methods
+
+### `Home`
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2 (Not yet)](https://img.shields.io/badge/API-Twitter%202%20(Not%20yet)-darkred?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public Home (Account account)
+```
+
+Creates `Home` for a given [`Account`](Account.md) and pulls posts.
+
+## Methods
```

Comment:
All of the other API definitions just have "inherited from" when the functions match. Is there a reason that these are redefined?

Reply:
I have to ask what do you refer to as "other API definitions" (I guess other documentations), but I "redefined" it to make clear that it is a different implementation from the parent class.

Reply:
Sorry, I mean "all of the other times in these API documents that a sub-class inherits a function have just had 'inherited from', but this one gets explicitly defined".

I wondered whether it was copy-and-paste that you forgot to change to "inherited" or whether it was for a reason. And apparently it's for a reason because they re-implement rather than just inheriting 🙂

Reply:
Actually, I looked at it and noticed that `get_posts` is technically just a getter for `post_list`, so I changed that to be inherited...